assets/js/accordion.js

```diff
@@ -1,6 +1,5 @@
-/*jslint browser, for */
-/*global window, ebLazyLoadImages, searchTerm, videoShow
-    locales, pageLanguage, console, Element, HTMLDocument,
+/* global window, ebLazyLoadImages, videoShow
```

Comment:
Are all of the changes to the JS just linting, or are there code changes that need closer review?

Reply:
Definitely just linting.